tests/test.py

```diff
@@ -309,7 +309,7 @@ def get_pregenerated_random_data():
         return f.read()
 
 
-def create_acrastruct_with_client_id(data, client_id='keypair1'):
+def create_acrastruct_with_client_id(data, client_id=TLS_CERT_CLIENT_ID_1):
```

Comment:
hm... it's strange that it works if some code relied on default `client_id`) because now at function declaration time client_id will be None everytime because of TLS_CERT_CLIENT_ID_1 re-initialized after interpreting and compiling function. What about to remove default value to force explicit declaring (now all code specify client_id explicitly) to not confuse in the future if something will be broken due to invalid value?